### DISCUSSION TITLE
v1.5.1

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#135] [Announcements]
- Commander CORE: Fixes incorrect sensor readings (#134).

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.5.1'>v1.5.1</a>.</em>

**Comment by bjongbloed** [comment]
Could you make the 50% minimum pump speed optional somehow? I have no issues with it running at 25% when idle and it's actually why I run this software. I'll be using 1.4.2 for now.

**Reply by EvanMulawski** [reply]
I can definitely do that - will be in a 1.6 beta release soon.
